### ISSUE TITLE
about aria-readonly=true

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax#1093]
Closely related to #939 

[From the User Group](https://groups.google.com/forum/#!topic/mathjax-users/bA_MsNjE2-8):

> [...] the screen reader reads "clickable on mouseover" after each inline term. This is because I am missing the aria-readonly=true attribute on the span with class=MathJax. 

And

> I did try putting the aria-readonly=true attribute on the MathML <math> tag, but the result is that it is applied to the incorrect span. The aria-readonly tag must be on the span with class="MathJax" in order to prevent the "clickable" issue. 

I'm not sure what the right approach is.

**Comment by pkra** [comment]
We're not sure what to do with this.

With the new AssistiveMML extension, the hidden MathML remains in the scope of the menu, thus `clickable on mouseover` would still occur. 

@jcsteh: you said it would be preferable to keep the menu linked to the hidden MathML. But iiuc NVDA will read "click ..." on every equation.

Theoretically, we could create a menu option to disable the menu, e.g., on all but the first math element. Not sure if that's really useful for AT users.

**Comment by pkra** [comment]
Should this drop from the v2.6 milestone?

**Comment by dpvc** [comment]
> Should this drop from the v2.6 milestone?

Yes, I think the other changes we have done may already have "resolved" this.
